src/components/Questionnaire/QuestionTypes/DiagnosisQuestion.tsx

```
@@ -588,52 +773,20 @@
-            </DropdownMenu>
-          </div>
-        </div>
-        <div className="grid grid-cols-2 gap-2 md:col-span-6 md:grid-cols-3 md:gap-4">
-          <div className="col-span-2 md:col-span-1">
-            <Label className="text-xs text-gray-500 md:hidden">
-              {t("date")}
-            </Label>
-            <Popover>
+        </TableCell>
+        <TableCell className="py-1 px-1">
+         <Popover>
```

ESLint

[error] Insert `·`

(prettier/prettier)